README.md

```diff
@@ -17,237 +17,106 @@ A distributed implementation of Redis compatible layer based on [TiKV](https://g
+|Streams| Not Supported Yet|
 
 ## FAQ
 
 [FAQ](https://github.com/distributedio/titan/issues?utf8=%E2%9C%93&q=+label%3A%22good+first+issue%22)
 
-## Commands
-
-### Connections
-- [x] auth 
-- [x] echo 
-- [x] ping
-- [x] quit 
-- [x] select 
-- [ ] swapdb, not supported
-
-### Transactions
-- [x] multi 
-- [x] exec
-- [x] discard
-- [x] watch
-- [x] unwatch
-
-### Server
-- [x] client list
-- [x] client kill
-- [x] client pause
-- [x] client reply 
-- [x] client getname
-- [x] client setname
-- [x] monitor
-- [x] debug object
-- [x] flushdb
-- [x] flushall
-- [x] time
-- [x] command
-- [x] command count
-- [x] command getkeys
-- [x] command info
-- [x] info
-- [ ] slowlog
-
-### Keys
-- [x] del 
-- [x] type
-- [x] exists
-- [x] expire
-- [x] expireat
-- [x] object 
-- [x] pexpire
-- [x] pexpireat
-- [x] ttl
-- [x] pttl
-- [x] randomkey
-- [ ] touch 
-- [x] keys
-- [x] scan
-- [ ] unlink
-
-### Strings
-
-- [x] get 
-- [x] set 
-- [x] mget 
-- [x] mset 
-- [x] strlen
-- [x] incr
-- [x] incrby
-- [x] decr
-- [x] decrby
-- [x] append
-- [ ] bitcount
-- [ ] bitfield
-- [ ] bitop
-- [ ] bitpos
-- [ ] getbit
-- [x] getrange
-- [x] getset
-- [x] incrbyfloat 
-- [ ] msetnx
-- [x] psetex
-- [ ] setbit
-- [x] setex
-- [x] setnx
-- [ ] setrange
-
-### List
-
-- [x] lrange
-- [x] linsert
-- [x] lindex
-- [x] llen
-- [x] lset
-- [x] lpush
-- [x] lpop
-- [x] lpushx
-- [ ] ltrim
-- [ ] lrem
-- [ ] rpop
-- [ ] rpoplpush
-- [x] rpush
-- [ ] rpushhx
-- [ ] blpop
-- [ ] brpop
-- [ ] brpoplpush
-
-### Hashes
-- [x] hset
-- [x] hget
-- [x] hgetall
-- [x] hdel
-- [x] hexists
-- [x] hincrby
-- [x] hincrbyfloat
-- [x] hkeys
-- [x] hlen
-- [x] hmget
-- [x] hmset
-- [x] hscan
-- [x] hsetnx
-- [x] hstrlen
-- [x] hvals
-
-### Sets
-
-- [x] sadd
-- [x] scard
-- [x] sdiff
-- [ ] sdiffstore
-- [x] sinter
-- [ ] sinterstore
-- [x] sismember
-- [x] smembers
-- [x] smove
-- [x] spop
-- [ ] srandmember
-- [x] srem
-- [x] sunion
-- [ ] sunionstore
-- [ ] sscan
-
-### Sorted Sets
-
-- [ ] bzpopmin
-- [ ] bzpopmax
-- [x] zadd
-- [x] zcard
-- [ ] zcount
-- [ ] zincrby
-- [ ] zinterstore
-- [ ] zlexcount
-- [ ] zpopmax
-- [ ] zpopmin
-- [x] zrange
-- [ ] zrangebylex
-- [ ] zrevrangebylex
-- [ ] zrangebyscore
-- [ ] zrank
-- [x] zrem
-- [ ] zremrangebylex
-- [ ] zremrangebyrank
-- [ ] zremrangebyscore
-- [x] zrevrange
-- [ ] zrevrangebyscore
-- [ ] zrevrank
-- [x] zscore
-- [ ] zunionstore
-- [ ] zscan
-
-### Geo
-
-- [ ] geoadd
-- [ ] geohash
-- [ ] geopos
-- [ ] geodist
-- [ ] georadius
-- [ ] georadiusbymember
-
-### hyperloglog
-
-- [ ] pfadd
-- [ ] pfcount
-- [ ] pfmerge
-
-### Pub/Sub
-
-- [ ] psubscribe
-- [ ] pubsub
-- [ ] publish
-- [ ] punsubscribe
-- [ ] subscribe
-- [ ] unsubscribe
-
-### Scripting
-
-- [ ] eval
-- [ ] evalsha
-- [ ] script debug
-- [ ] script exists
-- [ ] script flush
-- [ ] script kill
-- [ ] script load
-
-### Streams
-
-- [ ] xadd
-- [ ] xrange
-- [ ] xrevrange
-- [ ] xlen
-- [ ] xread
-- [ ] xreadgroup
-- [ ] xpending
+## Roadmap
+### Sprint #8
```

Comment:
The Sprint changes along with time, it would be out of date frequently. It is better to link to the Github project(which is our sprint).